#! /bin/sh /usr/share/dpatch/dpatch-run
## 12b_fix_a2ping_invocation.dpatch  <preining@logic.at>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix invocation of a2ping to get rid of magic

@DPATCH@
---
 texk/texlive/linked_scripts/a2ping/a2ping.pl |   10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

Index: texlive-bin-2009~svn13478/texk/texlive/linked_scripts/a2ping/a2ping.pl
===================================================================
--- texlive-bin-2009~svn13478.orig/texk/texlive/linked_scripts/a2ping/a2ping.pl	2009-05-26 11:00:07.000000000 +0200
+++ texlive-bin-2009~svn13478/texk/texlive/linked_scripts/a2ping/a2ping.pl	2009-05-27 22:16:58.000000000 +0200
@@ -1,11 +1,5 @@
-#! /bin/sh
-eval '(exit $?0)' && eval 'PERL_BADLANG=x;export PERL_BADLANG;: \
-;exec perl -x -S -- "$0" ${1+"$@"};#'if 0;
-eval 'setenv PERL_BADLANG x;exec perl -x -S -- "$0" $argv:q;#'.q+
-#!perl -w
-package Htex::a2ping;  $0=~/(.*)/s;unshift@INC,'.';do($1);die$@if$@;__END__+if !1;
-# This Perl script was generated by JustLib2 at Wed Apr 23 09:14:13 2003.
-# Don't touch/remove any lines above; http://www.inf.bme.hu/~pts/justlib
+#! /usr/bin/perl -w
+package Htex::a2ping;
 #
 # This program is free software, licensed under the GNU GPL, >=2.0.
 # This software comes with absolutely NO WARRANTY. Use at your own risk!
